client/src/Results/index.js

```diff
@@ -1,15 +1,47 @@
 import React, { Component } from "react";
+import Table, { TableBody, TableCell, TableHead, TableRow } from 'material-ui/Table';
+import Button from 'material-ui/Button';
+import Paper from 'material-ui/Paper';
 
 class Results extends Component {
   constructor(props) {
     super(props);
+
+    this.state = {
+      results: props.results || [{ name: "CMPSC 40", times: "M W 1:00 p.m - 1:50 p.m" }]
+    }
   }
 
   render() {
+    const { results } = this.state;
+
+    if(results.length === 0) {
+      return "No results found";
+    }
+
     return (
-      <div>
-        Place Results component here
-      </div>
+      <Paper>

```

Comment:
Maybe have a paper for the whole app and not every single component? A screenshot would be helpful here to see the bigger picture.